### DISCUSSION TITLE
Integration with reservations system

### DISCUSSION THREAD

**Discussion by nikdejan** [lucky-media/real-estate#3] [Q&A] [answered]
Hello,
Is it possible to use lucky-media/real-estate with reachweb/statamic-resrv or any other reservations addon?
I'm new to statamic and I want to develop online reservation platform for accommodation.

**Comment by lokmanm** [comment, marked as answer]
Hey @nikdejan,

Real Estate does not come with any reservation system. You are free to include anything you see fit in the Starter Kit.

If you are new to Statamic you might have trouble integrating resrv, but that's another topic. 

The starter kit does not limit you in any way.
